docs/our_work/template-project.md

```diff
@@ -4,7 +4,7 @@ summary: 'ONE LINE SUMMARY GOES HERE'
 origin: 'Skunkworks  (this is for legacy pages mainly, so we know where the work came from)'
 # tags: ['add your tags in here as a list (there is no limit) and remove the comment #','tag 2','tag 3']
 ---
-[comment]: <> When making your tags, it might be handy to see a list of the tags that have already been defined. Run `python utils/list-tags.py` from the terminal to print the existing tags (assuming you are in the root directory - update the path if not). You can also run it from VS Code by opening the file and clicking the "Run" button in the top right (but you'll need to have the Python extension installed to do it this way).
+[comment]: <> When deciding your tags, check out the [tag ontology](https://nhsengland.github.io/datascience/our_work/tags/) and try and choose a tag per category and not add any new tags. If you think new tags are required because the existing ones don't match your project, let someone in marketing know. 
```

Comment:
only thought is whether to change 'check out' to 'look at' (as check out has a particular specific technical meaning in the context of git repos)